tutorials/finetune_lora.md

```diff
@@ -34,7 +34,9 @@ The finetuning requires at least one GPU with ~24 GB memory (RTX 3090).
 
 This script will save checkpoints periodically to the folder `out/`.
 
-
+**Note**: LoRA can be applied to not only `query`, `key` or `value` matrices, but also to `projection`, `mlp` and classification `head`.</br>
```

Comment:
Did you mean to use this? https://github.com/orgs/community/discussions/16925, if so, it's missing the ">"

```suggestion
**Note**: LoRA can be applied to not only `query`, `key` or `value` matrices, but also to `projection`, `mlp` and classification `head`.</br>
```

Also I suggest avoiding "br" to break lines. You can just add newlines

Reply:
Yes, initially wanted to do this, but later decided that it "attracts" too much attention.
Given that it's actually an important information - I'll add ">" sign.

I prefer to use breaklines sometimes when I want the text to start from a new line and to not have too much spacing between lines, so it looks like a single block of text.
But I agree, since breaklines are not used anywhere else for the consistency I'll remove them.
Plus they don't work with ">" sign anyway :)